package.json

```diff
@@ -15,7 +15,8 @@
     "lint-css": "stylelint 'public/*.css' '.storybook/*.css' 'src/**/*.svelte' 'stories/**/*.svelte'",
     "storybook": "start-storybook -p 6006 -s ./",
     "build-storybook": "build-storybook -s ./",
-    "test": "npm run test-addon && npm run test-support-library"
+    "test": "npm run test-addon && npm run test-support-library",
+    "integration-test": "npm build && node tests/core-addon/selenium/core_addon.js"
```

Comment:
Can we retain the same style of the other test commands? e.g. `test-integration`?

Please also note that this command is failing on CI:

> npm WARN build `npm build` called with no arguments. Did you mean to `npm run-script build`?

It should be `npm run build && ...`

Reply:
oops I messed that latter part up moving from circle config to our `package.json`, sorry about that :/ corrected the name too.